zulipterminal/cli/run.py

```diff
@@ -339,12 +341,38 @@ def main(options: Optional[List[str]]=None) -> None:
         else:
             theme_to_use = zterm['theme']
 
+        if(zterm['footlinks'][1] == ZULIPRC_CONFIG
+           and zterm['maximum-footlinks'][1] == ZULIPRC_CONFIG):
+            exit_with_error(
+                error_message='Footlinks property is not allowed '
+                'alongside maximum-footlinks',
+                helper_text='',
+                error_code=1)
+
+        elif(zterm['maximum-footlinks'][1] == ZULIPRC_CONFIG
+             and int(zterm['maximum-footlinks'][0]) < 0):
+            exit_with_error(
+                error_message='Minimum value allowed '
+                'for maximum-footlinks is 0',
+                helper_text='',
+                error_code=1)
+
+        elif zterm['footlinks'][1] == ZULIPRC_CONFIG:
```

Comment:
This and the `elif` above can both be just `if`, as the earlier blocks both have an exit condition in them.